src/use-auth.test.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { MockedProvider } from '@apollo/client/testing';
+import { isApolloError } from '@apollo/client';
+import useAuth, { AuthProvider } from './use-auth';
+import QUERY_USERS from './graphql/queries';
+import REGISTER from './graphql/mutations';
+
+function TestComponent() {
```

Comment:
You shouldn't need to create a component to test the hook, you should test the existing pages that already consume that hook.
So basically you need to move those tests to the login and register pages instead: `src/Login/index.test.js` and `src/Register/index.test.js`